BusinessLogicLayer/seatsBLL.cs

```diff
@@ -20,14 +20,24 @@ public IEnumerable<Seats> GetAllSeats()
             return _seat.GetAllSeats();
         }
 
+        public void Insert(Seats seat)
+        {
+            _seat.Insert(seat);
+        }
+
+        public void Update(Seats seat)
+        {
+            _seat.Update(seat);
```

Comment:
this is not enough: 
 should get the existing seat from db (DAL) and then change it with the values from updatedSeat and then save to db.
 i.e.
public void Update(int id, Seat updatedSeat)
{
    var existing = _seat.GetSeatById(id);
    existing.SeatNumber = updatedSeat.SeatNumber;
   .. and all the fields that we want to be changed ...
    existing.Status = updatedSeat.Status;
    _seat.Update(existing);
}